src/main/java/com/cgdp/library/validation/ValidationUtility.java

```diff
@@ -0,0 +1,53 @@
+package com.cgdp.library.validation;
+
+import java.time.LocalDate;
+import java.util.function.Supplier;
+
+public class ValidationUtility {
+
+    public static Long required(String paramName, Long value) {
+        validate(() -> value == null, "%s must not be null", paramName);
+        return value;
+    }
+
+    public static String requiredNotBlank(String paramName, String value) {
+
+        validate(() -> value == null, "%s must not be blank", paramName);
+        return value;
+    }
+
+    public static LocalDate validatePublicationYear(String paramName, LocalDate value) {
+
+        validate(() -> value.isAfter(LocalDate.now()), "Invalid %s", paramName);
+        return value;
+    }
+
+    public static String validateISBN(String paramName, String value) {
```

Comment:
We should have specific class to do this since this is more complex validation, this method should call that class

And let's rename it to `requiredValidIsbn`